src/Core/Domain/Order/QueryResult/OrderProductForViewing.php

```diff
@@ -88,6 +88,21 @@ class OrderProductForViewing
      */
     private $orderDetailId;
 
+    /**
+     * @var string
+     */
+    private $amountRefund;
+
+    /**
+     * @var int
+     */
+    private $quantityRefunded;
+
+    /**
+     * @var string
+     */
+    private $amountRefundable;
```

Comment:
What's the difference between `amountRefund` and `amountRefundable` are they both formatted with currency symbol?

Reply:
in the handler amountRefound doesn't seem formatted

Reply:
amountRefundable is a float, containing the max value that can still be refunded for a product, contrary to amountRefund, it is not intended to be displayed as is, it is used for calculations.

amountRefund is indeed a string, build via `$this->locale->formatPrice(`